lib/l10n/intl_messages.arb

```diff
@@ -4071,5 +4065,11 @@
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
+  },
+  "messageWarningDialogWhenExpiredOIDCTokenAndReconnection": "Your session expired. We need to take you back to the login page in order to refresh it. You might want to save the email you are currently editing before we do so.",
```

Comment:
how about 
```suggestion
  "messageWarningDialogWhenExpiredOIDCTokenAndReconnection": "Your session has expired. We’ll try to save your draft before taking you to the login page, but please copy any important content to be safe.",
```

Reply:
This is pretty good for user guidance. Updated it.